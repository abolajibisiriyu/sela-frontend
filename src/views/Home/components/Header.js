import React from "react";

import HomeHeader from "../../../blocks/HomeHeader";
import HomeNavbar from "./Navbar";
import H4 from "../../../elements/H4";
import P from "../../../elements/P";

import chevronIcon from "../../../assets/images/expand-icon.svg";

function Header(props) {
  return (
    <HomeHeader>
      <HomeNavbar />
      <HomeHeader.Section>
        <H4 className="big-size" textAlign="center">
          Keep track of development <br /> projects near you
        </H4>
        <P className="big-size" textAlign="center">
          The Sela Platform enables stakeholders share real time information{" "}
          <br />
          on the state of infrastructure projects to ensure transparent delivery
        </P>
        <HomeHeader.Link onClick={props.executeScroll}>
          <span>SEE PROJECTS</span>
          <span>
            <img src={chevronIcon} alt="scroll down" />
          </span>
        </HomeHeader.Link>
      </HomeHeader.Section>
    </HomeHeader>
  );
}

export default Header;
