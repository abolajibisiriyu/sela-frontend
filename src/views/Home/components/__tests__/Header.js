import React from "react";
import renderer from "react-test-renderer";

import Header from "../Header";

describe("Header Component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
