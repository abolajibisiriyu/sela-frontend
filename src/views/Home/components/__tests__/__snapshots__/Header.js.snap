// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly 1`] = `
<header
  className="sc-htpNat Vdir"
>
  <nav
    className="sc-ifAKCX dLTBno"
  >
    <a
      className="sc-bxivhb bZRlOK"
      href="#"
    >
      SELA
    </a>
  </nav>
  <div
    className="sc-bwzfXH cEfGpb"
  >
    <h4
      className="big-size sc-EHOje knDRfy"
    >
      Keep track of development 
      <br />
       projects near you
    </h4>
    <p
      className="big-size sc-bZQynM lolnPI"
    >
      The Sela Platform enables stakeholders share real time information
       
      <br />
      on the state of infrastructure projects to ensure transparent delivery
    </p>
    <a
      className="sc-bdVaJa cXUSDU"
    >
      <span>
        SEE PROJECTS
      </span>
      <span>
        <img
          alt="scroll down"
          src="expand-icon.svg"
        />
      </span>
    </a>
  </div>
</header>
`;
